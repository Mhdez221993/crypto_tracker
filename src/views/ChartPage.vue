<template>
  <div id="chart">

    <div id="mydiv" v-on:click="dragElement">
      <div id="mydivheader">
        <ChartComponent />
      </div>
    </div>

  </div>
</template>

<script>

import ChartComponent from '../components/ChartComponent.vue'
import dragger from '../helpers/dragger'

export default {
  name: 'ChartPage',

  components: {
    ChartComponent,
  },

  methods: {
    dragElement: dragger
  },

  data() {
    return {
    }
  }
}

</script>