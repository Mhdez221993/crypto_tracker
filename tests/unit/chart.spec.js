// Components
// Utilities
import ChartPage from '@/views/ChartPage.vue'
import { createLocalVue, mount } from '@vue/test-utils'
import Vuetify from 'vuetify'



describe('ChartPage.vue', () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  it('should have a custom title and match snapshot', () => {
    const wrapper = mount(ChartPage, {
      localVue,
      vuetify,
    })

    expect(wrapper.html()).toMatchSnapshot()

    // We could also verify this differently
    // by checking the text content
    const title = wrapper.find('#mydivheader')

    expect(title.text()).toBe('Click here to move')
  })
})