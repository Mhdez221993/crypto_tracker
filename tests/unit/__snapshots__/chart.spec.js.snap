// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartPage.vue should have a custom title and match snapshot 1`] = `
<div class="chart">
  <h1>Chart</h1>
  <div id="mydiv">
    <div id="mydivheader">Click here to move</div>
  </div>
</div>
`;
